pep-0659.rst

```diff
@@ -44,7 +44,7 @@ Both academic research and practical implementations have shown that a fast inte
 
 Typical optimizations for virtual machines are expensive, so a long "warm up" time is required 
 to gain confidence that the cost of optimization is justified.
-In order to get speed-ups rapidly, without noticable warmup times,
+In order to get speed-ups rapidly, without noticeable warm-up times,
```

Comment:
"warmup" seems fine

Reply:
True, but codespell prefers **warm-up** and so do [Merriam-Webster](https://www.merriam-webster.com/dictionary/warmup) and [The Oxford Advanced Learner's Dictionary](https://www.oxfordlearnersdictionaries.com/definition/english/warm-up_2).
* [156,000,000 Google hits](https://www.google.com/search?q="warm-up") for **warm-up**
* [41,500,000 Google hits](https://www.google.com/search?q=warmup) for **warmup**

Reply:
I'd prefer to leave the original author's wording unless it's clearly wrong. English spelling has some variation.

Similarly, you changed "reenable" to "re-enable" at some point. That can also stay as "reenable".

Reply:
OK, I'll keep it as it is.

As for re-enable, it's probably my wrong.  I seem to recall it is a citation of a mail and shouldn't be changed. I'll look into it and fix it.

Reply:
That said, PEP 0659 itself is inconsistent:
> "warm up" time
> warmup times

Reply:
Fixed.